src/scripts/createTransaction.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * A script to create a transaction recording a new IPFS block Multihash in Stellar.
+ */
+
+import RahaStellar from '../RahaStellar';
+
+async function main() {
+    if (process.argv.length !== 5) {
+        console.log('Usage is: node uploadBlock.ts [isTest(y/n)] [multiHash] [secretKey]')
+        process.exit(1);
+    } else {
+        const isTest = process.argv[2] == 'y';
```

Comment:
Always use === unless you really mean ==, which is almost never. We should configure a linter to tell you that